--- grdesktop-0.23/grdesktop.desktop.orig	2003-11-12 15:47:22.000000000 +0100
+++ grdesktop-0.23/grdesktop.desktop	2005-05-14 22:01:05.000000000 +0200
@@ -1,7 +1,9 @@
 [Desktop Entry]
 Name=Remotedesktop Client
+Name[pl]=Klient zdalnego pulpitu (RDP)
 Comment=Connect to a remote Windows Terminal-Server
 Comment[de]=Verbindet zu einem entfernten Windows Terminal-Server
+Comment[pl]=Łączy ze zdalnym Windows Terminal Server
 TryExec=grdesktop
 Exec=grdesktop
 Icon=grdesktop.png
@@ -9,8 +11,9 @@
 Type=Application
 Version=1.0
 Actions=Full;
-Categories=Application;Network;X-Red-Hat-Base;
+Categories=GTK;GNOME;Network;RemoteAccess;
 Encoding=UTF-8
 
 [Desktop Action Full]
 Exec=grdesktop
+# vi: encoding=utf-8
