--- grdesktop-0.23/grdesktop.desktop.orig	2004-10-20 16:31:05.853679752 +0200
+++ grdesktop-0.23/grdesktop.desktop	2004-10-20 16:38:01.000000000 +0200
@@ -1,16 +1,19 @@
 [Desktop Entry]
 Name=Remotedesktop Client
+Name[pl]=Klient zdalnego pulpitu (RDP)
 Comment=Connect to a remote Windows Terminal-Server
 Comment[de]=Verbindet zu einem entfernten Windows Terminal-Server
+Comment[pl]=Łączy ze zdalnym Windows Terminal Server
 TryExec=grdesktop
 Exec=grdesktop
-Icon=grdesktop.png
+Icon=grdesktop/icon.png
 Terminal=false
 Type=Application
 Version=1.0
 Actions=Full;
-Categories=Application;Network;X-Red-Hat-Base;
+Categories=GTK;GNOME;Network;RemoteAccess;
 Encoding=UTF-8
 
 [Desktop Action Full]
 Exec=grdesktop
+# vi: encoding=utf-8
